app/assets/javascripts/contacts.js

```diff
@@ -4,18 +4,48 @@ var contentType = "application/json";
 var url = "http://localhost:5100/api/contacts";
 var storedContacts = {};
 
+var hideFeedback = function() {
+  $("[data-role='feedback']").hide();
+  $("[data-role='feedback']").remove("p");
+  $("[data-role='feedback']").removeClass("flash-success flash-error");
+}
+
+var showError = function(method) {
+  $("[data-role='feedback']").addClass("flash-error");
+  $("[data-role='feedback']").html(
+    "<p>Error: " + method + " was unsuccessful.</p>"
+  );
+  $("[data-role='feedback']").show();
+}
+
+var showSuccess = function(contact, method) {
+  hideFeedback();
+  method_msg = {
+    "PATCH": "Updated",
+    "POST": "Created"
+  }
+
+  $("[data-role='feedback']").addClass("flash-success");
+  $("[data-role='feedback']").html(
+    "<p>" + method_msg[method] + " " + contact["name"] + "</p>"
```

Comment:
Identifier 'method_msg' is not in camel case.<br>['name'] is better written in dot notation.<br>'method_msg' is not defined.